docs/src/appendix/cast/account/create.md

```diff
@@ -6,6 +6,7 @@ which is `~/.starknet_accounts/starknet_open_zeppelin_accounts.json` by default.
 
 ## Required common arguments - passed by CLI or specified in Scarb.toml
 
+* [`accounts-file`](../common.html#--accounts-file--f-path_to_accounts_file)
```

Comment:
This has a default value, so is not required :)

Reply:
Now that I see it, the flag should probably be passed to main sncast, and not to account (create|deploy) right?